dgtized/stack-mitosis
# Clone tags & attributes to copies

Ensures that the newly cloned target environment copies over the tags and attributes like backup windows & maintenance windows from the original target environment.

Previously this just reverted to the RDS defaults (or automatically copied from the source environment).

Tested on example databases using engines:

Other changes:

 - Add clj-kondo linting to test runner
 - Upgrade dependencies & test runner
 - Add in a presentation showing some of the basic intent of the tool
 - Test out dsl for adding tags and ensure test environment *has* tags to copy
 - Expand README docs to show overall process and document how attributes are cloned from target stack and not from the source stack.
